### PR TITLE
Added BottomSheet demos to the Material Gallery

### PR DESCRIPTION
**Pull request by HansMuller**

### CODE REVIEW

File: examples/material_gallery/lib/demo/persistent_bottom_sheet_demo.dart [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+        )
+      )
+    );
+  }
+}
+
+final WidgetDemo kPersistentBottomSheetDemo = new WidgetDemo(
+  title: 'Persistent Bottom Sheet',
+  routeName: '/persistentBottomSheet',
+  floatingActionButtonBuilder: (_) => _buildFloatingActionButton(),
```
Comment (Hixie) [inline examples/material_gallery/lib/demo/persistent_bottom_sheet_demo.dart]: why not inline the fab builder here?

**Comment by Hixie** [comment]
I'm not sure if this applies to material mobile apps, but on the Web, it's the prevailing opinion of accessibility experts that you shouldn't say "click here to foo", you should just say "foo". This would mean having buttons that say "Show bottom sheet" rather than "Tap here to show bottom sheet".

**Comment by Hixie** [comment]
LGTM

File: examples/material_gallery/lib/demo/persistent_bottom_sheet_demo.dart (hunk 1 above)
Comment (HansMuller) [inline examples/material_gallery/lib/demo/persistent_bottom_sheet_demo.dart]: Good point, done.

**Comment by HansMuller** [comment]
Leaving the "tap here" bits out sounds right to me, will do.

**Pull request merged**